<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
    <rosparam file="$(find nav_main)/config/odom/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find nav_main)/config/odom/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find nav_main)/config/odom/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav_main)/config/odom/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav_main)/config/odom/base_local_planner_params.yaml" command="load" />

     <rosparam file="$(find nav_main)/config/nav_obstacles_params.yaml" command="load" />
  </node>
  
</launch>
